### DISCUSSION TITLE
Recover the 3d vision of the bot in mp4 file or in real time stream

### DISCUSSION THREAD

**Discussion by papayedev** [PrismarineJS/mineflayer#2887] [General]
Hello 

Is it possible to get the video stream from the bot's camera? I would like to be able to extract some captures to use it in a CNN with PyTorch

**Comment by SonicandTailsCD** [comment]
Interesting! There is a way. Let's see if you're up for it:
1: Install the prismarine-viewer plugin
2: Load the plugin. 
3: After you're done and you run the bot, text should appear saying "Prismarine web browser running at *:3001". 
4: Install a recording software, like Bandicam. 
5: After it's installed, open your browser and type "http://localhost:3001"
6: Give it a minute or 2 - you need to let the browser render everything!
7: In Bandicam, select Record window or try to see if you can record freeform.
8: Select the browser window. You're done!
